src/planner/linux/multi.rs

```diff
@@ -30,6 +30,15 @@ impl Planner for LinuxMulti {
             return Err(Error::NixOs.into());
         }
 
+        // For now, we don't try to repair the user's Nix install or anything special.
+        if let Ok(_) = tokio::process::Command::new("nix")
```

Comment:
Does this work even if the `nix-command` feature is not enabled / is it guaranteed that we have `nix-command` enabled at this point?

If not, maybe it should become `if !(nix || nix-env)` (since `nix-env`, while it may some day go away, is what some places use to check for Nix, and even issues on the Nix repo request the output of `nix-env --version` to tell what version of Nix is being used).

Also, rather than just the bare command, it may be better to add `--version` or `--help` so that there there are no unintended side-effects (e.g. if some day `nix` by itself does something, rather than erroring as it does now).

Reply:
Ya, great catch. Thanks!

Reply:
I guess also `--extra-experimental-features nix-command` would also suffice in making this work without the need for `nix-env`.

Reply:
I like `nix-env` for this.